app/frontend/src/styles/application/govuk-error-summary.scss

```diff
@@ -0,0 +1,8 @@
+.govuk-error-summary__list {
```

Comment:
bit nit picky but put classes on things and use that selector to apply styles. ok its pretty obvious what an li inside a selector called `__list` is but its slightly less clear for the a below. using classes for stling is a good habit to be in, thats why i mention it